apps/fl-exp-simple/package.json

```diff
@@ -14,12 +14,13 @@
   "devDependencies": {
     "@aws-cdk/aws-appsync-alpha": "2.44.0-alpha.0",
     "@functionless/ast-reflection": "^0.3.1",
+    "@functionless/aws": "*",
     "@functionless/aws-constructs": "*",
     "@functionless/fl-exp": "*",
     "@functionless/language-service": "^0.0.4",
     "@functionless/register": "*",
     "@types/jest": "^29.0.3",
-    "@types/node": "^18.7.18",
+    "@types/node": "^18.0.0",
```

Comment:
nit: this version should reflect our minimum engine target (ex: `"^16"`).

If we allow node types to be higher, then we may use features not yet in consumer's node version.

Reply:
Moved them all to `^16`